packages/blocks/src/page-block/default/selection-manager/selection-manager.ts

```diff
@@ -36,6 +36,7 @@ import {
   isEmbed,
   isImage,
   isInsidePageTitle,
+  isInTitleBlock,
```

Comment:
Actually we don't have a title block, it's just the title field in the page block. And can we reuse the `isInsidePageTitle` method instead of adding a new one?

Reply:
ok, I didn't notice there already has one method